jina/peapods/runtimes/gateway/grpc/__init__.py

```diff
@@ -1,25 +1,24 @@
 import os
+from typing import AsyncGenerator
 
 import grpc
 
-from ..prefetch import PrefetchMixin
+from ..prefetch import PrefetchCaller
 from ...zmq.asyncio import AsyncNewLoopRuntime
 from ....zmq import AsyncZmqlet
-from .....logging.logger import JinaLogger
 from .....proto import jina_pb2_grpc
 
 __all__ = ['GRPCRuntime']
 
+from .....types.message import Message
 
-class GRPCPrefetchCall(PrefetchMixin, jina_pb2_grpc.JinaRPCServicer):
+
+class GRPCPrefetchCall(PrefetchCaller, jina_pb2_grpc.JinaRPCServicer):
```

Comment:
Why inheriting it and not wrapping it? I am OK with doing, but would like to see a good argument for it.

Reply:
Ah I think I got the argument. We don't have to handle down the `close` call. Nevermind.